shortcode "gallery" not found in GitHub Actions

```
Run hugo --minify
  hugo --minify
  shell: /bin/bash -e {0}
Building sites … Total in 27 ms
Error: Error building site: "/home/runner/work/segurvita.github.io/segurvita.github.io/content/home/gallery/index.md:12:1": failed to extract shortcode: template for shortcode "gallery" not found
##[error]Process completed with exit code 255.
```

--- a/.github/workflows/gh-pages.yml
+++ b/.github/workflows/gh-pages.yml
@@ -18,6 +18,9 @@ jobs:
           hugo-version: '0.62.2'
           extended: true
 
+      - name: Install academic theme
+        run: git submodule update --init --recursive
+
       - name: Build
         run: hugo --minify
 

--- a/.vscode/cspell.json
+++ b/.vscode/cspell.json
@@ -47,6 +47,7 @@
         "shortname",
         "STYLY",
         "Subdir",
+        "submodule",
         "TIMEMACHINE",
         "youtube",
         "Zepping"

